CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ Features:
   * adds support for `pagination links` at top level of JsonApi adapter [@bacarini]
   * adds extended format for `include` option to JsonApi adapter [@beauby]
   * adds support for wildcards in `include` option [@beauby]
+  * adds support for nested associations via the `include` key on the render call in controllers [@NullVoxPopuli, @beauby]
```

Comment:
This description is slightly misleading because 1. it is not constrained to controllers and 2. This functionality was already available in the JsonApi adapter.